src/index.html

```diff
@@ -1,14 +1,33 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="header">
+      <div class="headerLogo">
+        <img src="images/logo.png" alt="">
+      </div>
+      <nav class="headerNav">
+        <a href="#" class="navLink is-active">apple</a>
+        <a href="#" class="navLink">samsung</a>
+        <a href="#" class="navLink">смартфоны и телефоны</a>
+        <a href="#"
+           class="navLink"
+           data-qa="hover"
+      >ноутбуки и компьютеры</a>
```

Comment:
```suggestion
        <a href="#"
           class="navLink"
           data-qa="hover"
        >
          ноутбуки и компьютеры
        </a>
```
Поправь форматирование